### PR TITLE
 Store board-info.txt used for Firmware Assert in DT & convert dtb to dts

### PR DESCRIPTION
**Pull request by ShivamKumarJha**

### CODE REVIEW

File: dumpiara_v5_serv.sh [hunk 1]
```diff
@@ -57,6 +57,14 @@ for p in $PARTITIONS; do
         rm -rf $p\_
 done
 
+# board-info.txt
+find . -type f \( -name "NON-HLOS.bin" -o -name "modem" \) -exec strings {} \; | grep "QC_IMAGE_VERSION_STRING=MPSS.AT." | sed "s|QC_IMAGE_VERSION_STRING=MPSS.AT.|require version-baseband=|g" >> board-info.txt
```
Comment (erfanoabdi) [inline dumpiara_v5_serv.sh]: there's only modem, NON-HLOS.bin will be converted to modem

Comment (ShivamKumarJha) [reply dumpiara_v5_serv.sh]: i forgot about Firmware_extractor script. Ive force pushed with suggested change

File: dumpiara_v5_dropin.sh [hunk 2]
```diff
@@ -49,6 +49,15 @@ for p in system vendor modem cust odm oem; do
         sudo umount $p\_ &>/dev/null #unmount
         rm -rf $p\_
 done
+
+# board-info.txt
+find . -type f \( -name "NON-HLOS.bin" -o -name "modem.img" \) -exec strings {} \; | grep "QC_IMAGE_VERSION_STRING=MPSS.AT." | sed "s|QC_IMAGE_VERSION_STRING=MPSS.AT.|require version-baseband=|g" >> board-info.txt
+find . -type f \( -name "tz.mbn" -o -name "tz.img" \) -exec strings {} \; | grep "QC_IMAGE_VERSION_STRING" | sed "s|QC_IMAGE_VERSION_STRING|require version-trustzone|g" >> board-info.txt
```
Comment (erfanoabdi) [inline dumpiara_v5_dropin.sh]: firmware_extractor updated, now only tz.img can be found

File: dumpiara_v5_serv.sh [hunk 3]
```diff
@@ -57,6 +57,14 @@ for p in $PARTITIONS; do
         rm -rf $p\_
 done
 
+# board-info.txt
+find . -type f -name "modem.img" -exec strings {} \; | grep "QC_IMAGE_VERSION_STRING=MPSS.AT." | sed "s|QC_IMAGE_VERSION_STRING=MPSS.AT.|require version-baseband=|g" >> board-info.txt
+find . -type f \( -name "tz.mbn" -o -name "tz.img" \) -exec strings {} \; | grep "QC_IMAGE_VERSION_STRING" | sed "s|QC_IMAGE_VERSION_STRING|require version-trustzone|g" >> board-info.txt
```
Comment (erfanoabdi) [inline dumpiara_v5_serv.sh]: firmware_extractor updated, now only tz.img can be found

Comment (ShivamKumarJha) [reply dumpiara_v5_serv.sh]: force pushed again

**Comment by erfanoabdi** [comment]
noice, i'll test and push

**Comment by erfanoabdi** [comment]
find: ‘rename’: No such file or directory

**Comment by erfanoabdi** [comment]
if you can rewrite it using mv i appreciate it

**Comment by erfanoabdi** [comment]
merged

**Comment by ShivamKumarJha** [comment]
> if you can rewrite it using mv i appreciate it
I guess like this?
https://github.com/ShivamKumarJha/dumpyara/commit/af94d830011b3ab33c8d3d4bf5eb6ab23242a773

**Comment by erfanoabdi** [comment]
oh thanks i thought you are not going to do it, yes that's good 
if it works make another PR and remove the `rename` dependency from readme, i'll check again

**Comment by ShivamKumarJha** [comment]
> oh thanks i thought you are not going to do it, yes that's good
> if it works make another PR and remove the `rename` dependency from readme, i'll check again

Cool
done

**Pull request closed without merging**